catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -221,11 +229,32 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
 
         Map<String, List<Outer.Validator>> validators = new HashMap<>();
         for (Map.Entry<String, Object> entry : ((Map<String, Object>) root.get(VALIDATORS_PROPERTY)).entrySet()) {
```

Comment:
Might be able to make this a bit cleaner by doing `((Map<String, Object>)  root.get(VALIDATORS_PROPERTY)).forEach((attributeName, value) -> ...);`